packages/mui-material/src/FilledInput/FilledInput.js

```diff
@@ -140,13 +148,28 @@ const FilledInputInput = styled(InputBaseInput, {
   paddingRight: 12,
   paddingBottom: 8,
   paddingLeft: 12,
-  '&:-webkit-autofill': {
-    WebkitBoxShadow: theme.palette.mode === 'light' ? null : '0 0 0 100px #266798 inset',
-    WebkitTextFillColor: theme.palette.mode === 'light' ? null : '#fff',
-    caretColor: theme.palette.mode === 'light' ? null : '#fff',
-    borderTopLeftRadius: 'inherit',
-    borderTopRightRadius: 'inherit',
-  },
+  ...(!theme.vars && {
+    '&:-webkit-autofill': {
+      WebkitBoxShadow: theme.palette.mode === 'light' ? null : '0 0 0 100px #266798 inset',
```

Comment:
Maybe few more use-cases for the `unset` CSS var :)

Reply:
I think we should not put them in theme tokens yet. These are not directly palette tokens. There are only 2 components that contain this logic `FilledInput` and `OutlinedInput`, so I think let's wait for feedback from the community first.